user.c

```diff
@@ -0,0 +1,228 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+typedef struct
+{
+    int userID;
+    char userName[50];
+    int userAge;
+} Person;
+
+int checkUserID(int id)
+{
+    FILE *fp = fopen("users_data.txt", "r");
+    if (!fp)
+    {
+        // printf("Unable to open the file.\n");
+        return -1;
+    }
+    Person tempUser;
+    while (fscanf(fp, "%d,%49[^,],%d", &tempUser.userID, tempUser.userName, &tempUser.userAge) == 3)
+    {
+        if (tempUser.userID == id)
+        {
+            fclose(fp);
+            return 1;
+        }
+    }
+    fclose(fp);
+    return 0;

```

Comment:
Avoid multiple return statements in single function. The function should have one entry and one exit.